composer.json

```diff
@@ -53,34 +53,34 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-mongodb": "*",
         "container-interop/container-interop": "^1.2.0",
-        "doctrine/doctrine-module": "^5.0.0",
-        "doctrine/doctrine-laminas-hydrator": "^2.2.1 || ^3.0.0",
+        "doctrine/doctrine-module": "^5.1.0",
+        "doctrine/doctrine-laminas-hydrator": "^3.0.1",
         "doctrine/event-manager": "^1.1.1",
-        "doctrine/mongodb-odm": "^2.3.0",
-        "doctrine/persistence": "^2.2.3",
+        "doctrine/mongodb-odm": "^2.3.2",
+        "doctrine/persistence": "^2.4.0",
```

Comment:
What's the point in bumping `doctrine/persistence`? [ODM itself is fine with ^2.2](https://github.com/doctrine/mongodb-odm/blob/2.3.x/composer.json#L32)

Reply:
Why not? Since it is the same major there shouldn't be any issues in upgrading.

Reply:
Probably best done in a minor release, rather than a patch release

Reply:
Ok, I changed the base